app/components/ilios-navigation.hbs

```diff
@@ -198,6 +198,23 @@
           </LinkTo>
         </li>
       {{/if}}
+      <li>
```

Comment:
Feels to me like this should be above admin, personal opinion only.

Reply:
:heavy_check_mark: